cmd/monaco/download/download_configs.go

```diff
@@ -132,11 +138,11 @@ func (d DefaultCommand) DownloadConfigsBasedOnManifest(fs afero.Fs, cmdOptions d
 }
 
 func (d DefaultCommand) DownloadConfigs(fs afero.Fs, cmdOptions downloadCmdOptions) error {
-	a, errors := cmdOptions.auth.mapToAuth()
-	errors = append(errors, validateParameters(cmdOptions.environmentURL, cmdOptions.projectName)...)
+	a, errs := cmdOptions.auth.mapToAuth()
+	errs = append(errs, validateParameters(cmdOptions.environmentURL, cmdOptions.projectName)...)
```

Comment:
nit: could we rename `validateParameters` to make it clear it's specific to direct download? 
My initial though was that it should be part of the options.validate() function now, but I think it does not actually apply to the Manifest based download, right?

Reply:
This observation is in place. But as this part is out of scope, and I have no idea to propose a new name, I'll leave it as it is.